File: src/Tarea7-Unidad2-Java/Employee.java
import java.util.ArrayList;

public class Employee {
    private String name;
    private String lastName;
    private ArrayList<BankAccount> accounts;

    public Employee(String name, String lastName, ArrayList<BankAccount> accounts) {
        this.name = name;
        this.lastName = lastName;
        this.accounts = accounts;
    }

    public String getName() {
        return name;
    }

    public String getLastName() {
        return lastName;
    }

    public ArrayList<BankAccount> getAccounts() {
        return accounts;
    }

    public void viewEmployee() {
        System.out.println("Employee: " + name + " " + lastName);
        for (BankAccount account : accounts) {
            System.out.println("Account Type: " + account.getAccountType());
        }
    }

    public void viewEmployeeDetails() {
        System.out.println("Employee: " + name + " " + lastName);
        for (BankAccount account : accounts) {
            System.out.println("Account Number: " + account.getAccountNumber());
            System.out.println("Balance: $" + account.getBalance());
            System.out.println("Account Type: " + account.getAccountType());
        }
    }

}

File: src/Tarea7-Unidad2-Python/employee.py
class Employee:
    def __init__(self, name, last_name, accounts):
        self.name = name
        self.last_name = last_name
        self.accounts = accounts

    def view_employee(self):
        print(f"Employee: {self.name} {self.last_name}")
        for account in self.accounts:
            print(f"Account Type: {account.account_type}")

    def view_employee_details(self):
        print(f"Employee: {self.name} {self.last_name}")
        for account in self.accounts:
            print(f"Account Number: {account.account_number}")
            print(f"Balance: ${account.balance}")
            print(f"Account Type: {account.account_type}")
